plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/editparts/ConnectionEditPart.java

```diff
@@ -473,8 +473,15 @@ private FBNConnectionEndpointPolicy getConnectionEndPointPolicy() {
 	}
 
 	@Override
-	public void updateAnnotations(final GraphicalAnnotationModelEvent event) {
+	public void updateAnnotations(final GraphicalAnnotationModelEvent event) { // missing null check
```

Comment:
Please remove the stray comment:
```suggestion
	public void updateAnnotations(final GraphicalAnnotationModelEvent event) {
```